Source/Samples/Sections/Widgets/RangeSection.cs

```diff
@@ -0,0 +1,42 @@
+
+// This is free and unencumbered software released into the public domain.
+// Happy coding!!! - GtkSharp Team
+
+using Gtk;
+
+namespace Samples
+{
+    [Section(ContentType = typeof(Range), Category = Category.Widgets)]
+    class RangeSection : ListSection
+    {
+        static HScale hScale;
+        static VScale vScale;
```

Comment:
No point in keeping these static.

Reply:
cleaned up the reference from older docs to var syntax.